QSB/QSB.csproj.user

```diff
@@ -3,5 +3,6 @@
   <PropertyGroup>
     <GameDir>D:\EpicGames\OuterWilds</GameDir>
     <OwmlDir>C:\Users\Henry\AppData\Roaming\OuterWildsModManager\OWML</OwmlDir>
+    <ProjectView>ShowAllFiles</ProjectView>
```

Comment:
🤔

Reply:
was useful for me to find files VS had hidden by accident. like there was a copy of QuaternionHelper in the TransformSync namespace...